apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/ir/Fragment.kt

```diff
@@ -5,9 +5,10 @@ import com.apollographql.apollo.compiler.VisitorSpec.VISITOR_CLASSNAME
 import com.squareup.javapoet.ClassName
 import com.squareup.javapoet.FieldSpec
 import com.squareup.javapoet.TypeSpec
+import com.squareup.moshi.JsonClass
 import javax.lang.model.element.Modifier
 
-data class Fragment(
+data class ParsedFragment(
```

Comment:
As the `Fragment` and `Operation` IR classes contained absolute file paths, I had to duplicate the struct:

* `ParsedFragment` is used during parsing and contains the absolute file path for diagnostics purposes
* `ParsedFragment` is serialized and later used by codegen. It contains package name and name but no absolute file path.